backend/app/controllers/spree/admin/resource_controller.rb

```diff
@@ -222,7 +222,8 @@ def build_resource
   end
 
   def collection
-    return parent.send(controller_name) if parent?
+    return parent.send(controller_name) unless parent.nil?
```

Comment:
I feel like this _shouldn't_ happen. In what situations might it? Would a better solution be to change the implementation of `#parent?`?